### ISSUE TITLE
feature request: easy updates

### ISSUE THREAD

**Issue by magnetophon** [alexanderjeurissen/ranger_devicons#15]
Is there a way to keep this wonderful plugin up to date that doesn't involve manually checking this repo?

As a partial solution I suggest changing the line that is inserted to ``rc.conf`` to include where the plugin came from:
```
# a plugin that adds file glyphs / icon support to Ranger:
# https://github.com/alexanderjeurissen/ranger_devicons
default_linemode devicons
```

That way, when you update ranger, and diff ``rc.conf``, you are reminded to have a look here.

**Comment by alexanderjeurissen** [comment]
Hi @magnetophon Thanks so much for filing this issue, it has definitely come to my attention that this is indeed a limitation currently. One which I myself am not confronted with as seeing as the author of this plugin I have a copy of the repo laying around on all my machines. 

I'll implement your suggestion to include a comment for now, and think about how we can improve the update process.

**Comment by magnetophon** [comment]
Dank je wel!

**Comment by alexanderjeurissen** [comment]
I thought about this some more and I think that wrapping this plugin in a PyPy package could be a potential improvement, the only trick would be to get the files in the `.config/ranger` directory

resources for my own reference sake:

- http://peterdowns.com/posts/first-time-with-pypi.html
- https://github.com/pypa/pip/issues/1489

This would allow for the following installation flow:

`$ pip install ranger-devicons`
`$ ranger-devicons --install`

The second command would actually symlink the files to the ranger directory, so that a future `pip install --upgrade ranger-devicons` can pull in the latest version.

@magnetophon would a flow as outlined above meet you're expectations of a easier update/installation process ?

**Comment by magnetophon** [comment]
Sounds great, thanks!

**Comment by alexanderjeurissen** [comment]
Closed by #37

**Comment by magnetophon** [comment]
Cool, thanks!
